### DISCUSSION TITLE
Support for electron apps with Percy + Playwright

### DISCUSSION THREAD

**Discussion by ggilchrist-ledger** [percy/cli#822] [Support]
Hi, 

I'm looking to add visual regression with Percy for our Electron/React Native application. We use Playwright to automate our UI testing and so far the `@percy/playwright` intergration has worked well, but trying to upload the snapshots results in: 

```
[percy] Encountered an error uploading snapshot: Swap Page with exchange ready
[percy] Error: Invalid URL, scheme must be http or https: file:///Users/...........
```

Judging by your docs Electron isn't supported yet. Can you confirm this and if there are any workarounds to be able to upload assets from an Electron app to Percy? I've seen a suggestion that I can host the `.webpack/index.html?theme=null` file on localhost and upload that but I'm not too sure how to go about it. Thanks

**Comment by Robdel12** [comment]
Hey @ggilchrist-ledger! I'm not sure exactly what needs to be done to make it work, but since Electron apps are HTML/CSS/JavaScript it _can_ work. Percy does require valid URLs though, so whatever is being snapshotted can't have a URL like `file:///Users/`. I believe that's largely up to how the electron app is built, but I'm not sure (I'm not too familiar with Electron)
